### DISCUSSION TITLE
youtube api

### DISCUSSION THREAD

**Discussion by lisacek** [eritislami/evobot#220] [Q&A] [answered]
Guys, how to fix this?
The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.

**Comment by ghost** [comment, marked as answer]
Make a new API key from:
[Google Developers Console](https://console.developers.google.com)
Just go there, 
Make a new project.
When it says making the project on right side,
And when it's done,
Click on View
After that click on credentials and click create a new API!
Copy that API and paste it in `config.json`.
Now go back to [Google Developers Console](https://console.developers.google.com)
and then go to library.
Go down until you find **Youtube Data V3 API**
Click on it and click **Enable**. Wait for it to finish.
Voila! You just got your new API!

**Reply by caeptnlaen** [reply]
I can choose at credintail between server and browser, wich one is the right
